server/src/main/java/com/cloud/network/security/SecurityGroupManagerImpl.java

```diff
@@ -1410,7 +1407,7 @@ public SecurityGroupVO getDefaultSecurityGroup(long accountId) {
     }
 
     @Override
-    public SecurityGroup getSecurityGroup(String name, long accountId) {
+    public SecurityGroupVO getSecurityGroup(String name, long accountId) {
```

Comment:
@weizhouapache, would it be better to still return `SecurityGroup` in the calling method and maybe add methods to the interface? We are adding coupling this way.

Reply:
done